t/router/radixtree-sni.t

```diff
@@ -313,6 +314,7 @@ sent http request: 62 bytes.
 received: HTTP/1.1 200 OK
 received: Content-Type: text/plain
 received: Connection: close
+received: Server: APISIX/(\d+)\.(\d+)\.(\d+)
```

Comment:
`(\d+)\.(\d+)\.(\d+)` -> `([\w\.]+)`

Reply:
There are two version numbers for the version number, such as 1.4 or 1.4.1 now.

Reply:
Should we update this to `([\d\.]+)(\d)$`
`1.4.1fd` and `1.4.1.` also passes `([\d\.]+)`